nbs/models.timemixer.ipynb

```diff
@@ -816,16 +816,8 @@
     "from neuralforecast import NeuralForecast\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.                    max_steps=100,
</pre><p>Could we increase the max step to 500 like in other models? Also gives users an idea that DL should ideally run for around 500, even 1000 steps to get good predictions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Thanks, changed to 500